src/components/geonode.jsx

```diff
@@ -134,12 +136,15 @@ class GeoNodeViewer extends React.Component {
         onRequestClose={this._handleRequestClose.bind(this)}
       />);
     }
-    let layerList = undefined;
+    let layerList, save = undefined;
     if(this.edit) {
       layerList = {
         sources: this.props.addLayerSources,
         allowUserInput: true
       };
+      if(this.props.server) {
+        save = (<div id='save-button' className='ol-save'><Save map={map} /></div>);
```

Comment:
maybe we shouldn't use ```ol-``` prefix for css classes here? Since that is used by OpenLayers?

For the popup it's done because openlayers has that class already.